core/templates/dev/head/services/UpgradedServices.ts

```diff
@@ -233,6 +238,7 @@ import { WrittenTranslationObjectFactory } from
   'domain/exploration/WrittenTranslationObjectFactory';
 import { WrittenTranslationsObjectFactory } from
   'domain/exploration/WrittenTranslationsObjectFactory';
+import { from } from 'rxjs';
```

Comment:
What is this import for?

Reply:
Sorry that was by mistake imported
I'll fix it.
Thanks